run.sh

```diff
@@ -50,23 +54,47 @@ if [ -f /share/tesla_ble_mqtt/private.pem ]; then
  delete_legacies
  exit 1
 fi
-if [ ${TESLA_VIN-} ]; then
- log_error "Using depecated configuration parameters --> Exiting."
- log_error "Fix config and restart. If you see this message again, please raise an issue"
+if [ ! -z ${TESLA_VIN} ]; then
```

Comment:
Simmons might thing I’m picky but the original isn’t quite standard, you can check with ChatGPT ;)